python/marvin/tools/mixins/mma.py

```diff
@@ -22,17 +22,19 @@
 from marvin.core.exceptions import MarvinError, MarvinMissingDependency, MarvinUserWarning
 from marvin.utils.db import testDbConnection
 from marvin.utils.general.general import mangaid2plateifu
-
-
+from sdss_access import os_windows
 try:
     from sdss_access.path import Path
 except ImportError:
     Path = None
 
 try:
-    from sdss_access import RsyncAccess
+    if os_windows:
+        from sdss_access import CurlAccess as Access
+    else:
+        from sdss_access import RsyncAccess as Access
```

Comment:
I commented on this in the `sdss_access` `curl` PR.  I think this kind of logic should be moved into `sdss_access` and handled internally to that package.  The only thing `marvin` or other software would need to do is something like `from sdss_access import Access`, for example.

Reply:
We started on the base class, but ran into some syntax issues, so @mstalbot will re-try himself.  I wasn't sure about this, however (and commented as such on that PR), until I read the above comment, so I now agree that it would be simpler for Marvin to just use an Access class which would default to RsyncAccess for posix and CurlAccess otherwise.

Reply:
Sounds good!